wsagent/che-core-api-languageserver/src/main/java/org/eclipse/che/api/languageserver/service/LanguageServiceUtils.java

```diff
@@ -31,6 +33,14 @@ public static String removePrefixUri(String uri) {
     return uri.startsWith(FILE_PROJECTS) ? uri.substring(FILE_PROJECTS.length()) : uri;
   }
 
+  public static List<String> removePrefixUri(List<String> uris) {
+    for (int i = 0; i < uris.size(); i++) {
```

Comment:
Better to return a new copy of the list. It's not obvious for the caller that this method modifies the list that is passed in. Allocating a new List is a cheap operation. "Premature optimization is the root of all evil".

Reply:
agree